freqtrade/rpc/telegram.py

```diff
@@ -221,20 +221,24 @@ def _format_buy_msg(self, msg: Dict[str, Any]) -> str:
                 msg['stake_amount'], msg['stake_currency'], msg['fiat_currency'])
         else:
             msg['stake_amount_fiat'] = 0
-        is_fill = msg['type'] == RPCMessageType.BUY_FILL
+        is_fill = msg['type'] in [RPCMessageType.BUY_FILL, RPCMessageType.SHORT_FILL]
         emoji = '\N{CHECK MARK}' if is_fill else '\N{LARGE BLUE CIRCLE}'
 
+        enter_side = ({'enter': 'Long', 'entered': 'Longed'} if msg['type']
+                      in [RPCMessageType.BUY_FILL, RPCMessageType.BUY]
+                      else {'enter': 'Short', 'entered': 'Shorted'})
         message = (
-            f"{emoji} *{msg['exchange']}:* {'Bought' if is_fill else 'Buying'} {msg['pair']}"
+            f"{emoji} *{msg['exchange']}:*"
+            f" {enter_side['entered'] if is_fill else enter_side['enter']} {msg['pair']}"
             f" (#{msg['trade_id']})\n"
             )
         message += f"*Enter Tag:* `{msg['enter_tag']}`\n" if msg.get('enter_tag', None) else ""
+        message += f"*Leverage:* `{msg['leverage']}`\n" if msg.get('leverage', None) else ""
```

Comment:
I don't think this test for leverage (neither here, nor when buying, nor in tests) is correct.

It's based on the assumption that a "regular" order (spot order) will have a leverage of `None`.
it will however have a leverage of 1 - which will resolve to True if checked as it is here.
I think this should check `msg.get('leverage', 1.0) != 1.0` ... obviously we'd want to also slightly adapt tests to account for this.

![Screenshot_20211230_172336](https://user-images.githubusercontent.com/5024695/147769774-ef411324-424b-44ed-916e-6bbac03e7886.png)


I've adapted/updated the code for this in the latest commit...